VIDEO_1_SCRIPT.md

```diff
@@ -55,23 +55,23 @@ E-commerce, Social media, cloud computing, mobility… Without a doubt, we are d
 
 Now that we have covered the origins and popularization of API, let's talk about the types of APIs.
 
-- ***Web services APIs***: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
+* ***Web services APIs***: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
 
     ***There are four types*** of Web services APIs:
 
-    •	***SOAP*** (Simple Object Access Protocol), ***a standard protocol for exchanging information and data in XML between two objects.***
+    *	***SOAP*** (Simple Object Access Protocol), ***a standard protocol for exchanging information and data in XML between two objects.***
 
-    •	***XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.***
+    *	***XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.***
 
-    •	***JSON-RPC, same as XML-RPC but in JSON format.***
+    *	***JSON-RPC, same as XML-RPC but in JSON format.***
 
-    •	***REST*** (Representational State Transfer), ***a software architecture for hypermedia systems on the World Wide Web; a REST API often uses the HTTP protocol.***
+    *	***REST*** (Representational State Transfer), ***a software architecture for hypermedia systems on the World Wide Web; a REST API often uses the HTTP protocol.***
 
 - ***Library-based APIs: These are APIs designed for a specific programming language. They are a means to reuse libraries from other software or other parts of the same software.***
 
 - ***Function APIs in operating systems: software programs are continuously interacting with operating systems (OS). In many cases, the way they do so is through APIs.*** Operating systems such as Windows have APIs that enable such communication between programs and the OS.
 
-## ***Benefits of APIs***
+## ***Benefits of Web APIs***
```

Comment:
kinda, don't know but somehow I think we need this clarification, maybe also some text should be in previous section that ok, there are 3 types of APIs we will not expand in training on Library and Function APIs

Reply:
Yes, it makes sense. I proposed a suggestion about this above.